app/dispatchers/command_dispatcher.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+from app.dispatchers.display_dispatcher import DisplayDispatcher
+from app.dispatchers.print_dispatcher import PrintDispatcher
+from app.dispatchers.serialize_dispatcher import SerializeDispatcher
+from app.entities.book import Book
+
+
+class CommandDispatcher:
+    RETURN_COMMANDS = ["serialize"]
+
+    ALLOWED_DISPATCHER = {
+        "display": DisplayDispatcher(),
+        "print": PrintDispatcher(),
+        "serialize": SerializeDispatcher()
+    }
```

Comment:
To follow the DIP more strictly, you could pass these dependencies in as arguments.

Reply:
Okey, thx